client/platform/desktop/frontend/components/ImportDialog.vue

```diff
@@ -38,6 +39,23 @@ export default defineComponent({
       argCopy.value.jsonMeta.fps = clientSettings.annotationFPS;
     }
 
+    // Set default attributes for a stereo calibration configuration
+    if (argCopy.value.jsonMeta.stereoConfigurationFile) {
+      argCopy.value.jsonMeta.attributes = {};
+
+      // Create x, y, z attributes
+      ['x', 'y', 'z'].forEach((attributeKey) => {
+        // ugly but necessary to avoid typescript error
+        (argCopy.value.jsonMeta.attributes as Record<string, Attribute>)[attributeKey] = {
+          belongs: 'detection',
+          datatype: 'number',
+          key: `detection_${attributeKey}`,
+          name: attributeKey,
+          values: [],
+        };
+      });
+    }
+
```

Comment:
should these attribute keys be more specific about being stereo3D attribute keys.  There are other users who do use attributes like this already.

Reply:
agree to update the naming. where is the best place to update the documentation about what attributes are "reserved"?

Reply:
does stereo3d_x, stereo3d_y, stereo3d_z looks ok to you?

Reply:
Yeah that would be perfect.  If you want to indicate that these attributes in combination with a stereoConfiguration file are reserved that could be added here: https://github.com/Kitware/dive/blob/main/docs/UI-Attributes.md

Reply:
Thibault opened a PR for VIAME as well here: https://github.com/VIAME/VIAME/pull/162